kubespawner/spawner.py

```diff
@@ -107,9 +125,27 @@ def events(self):
         )
 
 
+class MultiNamespaceEventReflector(MultiNamespaceResourceReflector):
+    """
+    MultiNamespaceEventReflector is the all-namespaces analog of
+    EventReflector.
+    """
+
+    kind = "events"
+    list_method_name = "list_event_for_all_namespaces"
```

Comment:
In general, what do you think about refactoring the reflectors to use `kind` to construct `list_method_name`. Do these all have the form `list_{kind}_for_all_namespaces` or `list_namespaced_{kind}` (if kind were singular, which it is not currently). Then we could just have two parameters: `kind` and `omit_namespace` to configure most values?

Reply:
Oooh, I like that.

Reply:
Done!